app/scripts/components/Modal/ShareUrl.jsx

```diff
@@ -13,14 +12,16 @@ class ShareUrl extends React.Component {
   }
 
   componentDidMount() {
-    this.props.getShortLink(this.props.url);
+    if (!this.props.iframe) {
+      this.props.getShortLink(this.props.url);
+    }
   }
 
   componentWillReceiveProps(nextProps) {
     const { url: nextUrl } = nextProps;
     const { url: prevUrl } = this.props;
 
-    if (nextUrl !== prevUrl) {
+    if (nextUrl !== prevUrl && !this.props.iframe) {
```

Comment:
Shouldn't you check `nextProps.iframe` instead?

Reply:
Yes, you are right